src/site/components/navigation-sidebar.html

```diff
@@ -23,7 +23,9 @@
 
 <nav {% if relatedPages != empty %}class="usa-width-one-fourth va-sidebarnav"{% endif %} id="va-detailpage-sidebar">
   <div>
-    <button type="button" aria-label="Close this menu" class="va-btn-close-icon va-sidebarnav-close"></button>
+    {% if hidesidenav === empty %}
```

Comment:
Is it possible to wrap the navigation-sidebar include in `detail-page.html` with this if statement instead? That would be cleaner, I think, unless it messes up the grid.